### DISCUSSION TITLE
<Table /> vertical headers

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [mantinedev/mantine#6924] [Feature requests]
It'd be nice to have an option for `<Table />` to show `<Table.Thead />` vertically:
![Screenshot 2024-10-01 at 02 05 33](https://github.com/user-attachments/assets/2248f863-fb26-41b0-b248-675b8d8986a3)

Currently all headers are horizontal:
![Screenshot 2024-10-01 at 02 08 44](https://github.com/user-attachments/assets/ff5c3e97-e24a-4184-9ca0-613971577fb0)

**Comment by rtivital** [comment]
`Table.Thead` cannot be shown vertically. Instead you can put <th /> elements inside rows of tbody. For example:

```html
<table>
  <tbody>
    <tr>
      <th>Row Header 1</th>
      <td>Data 1</td>
      <td>Data 2</td>
    </tr>
    <tr>
      <th>Row Header 2</th>
      <td>Data 3</td>
      <td>Data 4</td>
    </tr>
  </tbody>
</table>
```
